### ISSUE TITLE
Dots are not displayed on map when mouse over on other panels

### ISSUE THREAD

**Issue by varejjka** [pR0Ps/grafana-trackmap-panel#54]
Hi!
Could you please advise me what I do wrong?

<img width="742" alt="Снимок экрана 2021-02-28 в 00 05 29" src="https://user-images.githubusercontent.com/22656576/109400313-1ec1ce00-7959-11eb-85e1-cec753336d22.png">

**Comment by pR0Ps** [comment]
Duplicate of #50

A fix has already been released, just waiting on Grafana to publish it (see #52)

**Comment by zolbarna** [comment]
Same things happened with me recently.

I still have a Grafana what I created recently (about a week old, and it's working) 

Since then I've created 2 new environment. (same distro, same steps) but I'm not able to get it work again. 

My 1 week old env is still works, and I can't find out the differences.  Your plugin is the same everywhere but the grafana is not:

RPI4, docker env, Grafana: v7.4.3 (010f20c1c8) - Not working
Virtual Ubuntu, docker env Grafana: v7.3.10 (43e9cd2a02) - Working
Virtual Ubuntu, docker env Grafana: v7.4.5 (8a2c78d3f8) - Not working

![image](https://user-images.githubusercontent.com/70371841/111895795-3bf83280-8a15-11eb-8abf-2cacbc399d49.png)

**Comment by zolbarna** [comment]
Ok.

Your 2.1.2 version will be fixing this issue, as soon as it will be available. I realized you released a fix for Grafana 7.4. Thanks for your hard work anyway.

**Comment by UlrichThiess** [comment]
Grafana 7.5 on Windows
trackmap-panel 2.1.2

It still won't work with new 'Time series' panel

**Comment by MatinF** [comment]
@UlrichThiess A bit off topic: I saw your dashboard, could you let me know what plugin and config you used to create the 'heading'/compass plugin?

![image](https://user-images.githubusercontent.com/26184427/115156736-0e3ef180-a086-11eb-9b73-4e4e9de00f99.png)

**Comment by UlrichThiess** [comment]
https://grafana.com/grafana/plugins/briangann-gauge-panel/

**Comment by pR0Ps** [comment]
Fixed by #77 in v2.1.3
